src/test/java/edu/harvard/iq/dataverse/util/MailUtilTest.java

```diff
@@ -98,25 +98,25 @@ public void testSubjectRejectFileAccess() {
     @Test
     public void testSubjectCreateDataset() {
         userNotification.setType(UserNotification.Type.CREATEDS);
-        assertEquals("LibraScholar: Your dataset has been created", MailUtil.getSubjectTextBasedOnNotification(userNotification, null));
+        assertEquals("LibraScholar: Dataset \"\" has been created", MailUtil.getSubjectTextBasedOnNotification(userNotification, null));
```

Comment:
Just to confirm - you only need empty quotes here because the test dataset doesn't have a display name?

Reply:
@qqmyers :
The method testSubjectCreateDataset() checks for the email subject after the dataset is created.  
Since Null object is passed from this method `MailUtil.getSubjectTextBasedOnNotification(userNotification, null)` ,  which results in empty displayname.  
Hence _notification.email.create.dataset.subject_ property from bundle.properties constructs like above.
notification.email.create.dataset.subject={0}: Dataset "{1}" has been created

Also the changes in MailUtil.java is made to handle these null objects in test cases.  Please let me know, if I misunderstood your comment.  Thanks